netcloud/scripts/test_netcloud_registration.py

```diff
@@ -0,0 +1,394 @@
+""" Simple script to poke at the router <-> NetCloud registration process. """
+
+import socket
+import ssl
+import json
+import functools
+import hashlib
+import threading
+import collections
+import netifaces
+import api.stream
+from api.base import RootException, Timeout, NotRegistered, Disabled, Unauthorized
+
+# Required: Add your NetCloud credentials (required for registering the router with NetCloud)
+_CID = "your-netcloud-user"
+_CPW = "your-netcloud-password"
+
+# Optional: Alternatively, use an existing NetCloud access token which you get from your rooted Cradlepoint device
+_TOKEN = {
+    "client_id": 844,
```

Comment:
not sure about client_id, maybe change it to a variable.

Reply:
Done.